docs/get-started/quick-start-guide.md

```diff
@@ -0,0 +1,145 @@
+>>>>>>> 2fa3e4e (first docs reorg)
+
+## Step 2: Install the Planet Earth Observation Toolkit
+
+Install with [pip](https://pip.pypa.io):
+
+```console
+$ pip install planet --user
+```
+
+The [--user](https://pip.pypa.io/en/stable/user_guide/#user-installs)
+flag is highly recommended for those new to pip.
+
+## Step 3: Get your API Key ready
+
+<<<<<<< HEAD
+Planet's APIs require an account for use. For more information on where to find your API key, or to sign up to get one, see [Get Your API Key](get-your-api-key.md).
+=======
+Planet's APIs require an account for use. For more information on where to find your API key, or get one if you don't already have one, see [Get Your API Key](get-your-api-key.md).
+>>>>>>> 2fa3e4e (first docs reorg)
+
+## Step 4: Check the toolkit version
+
+```console
+$ planet --version
+```
+
+<<<<<<< HEAD
+You should be on some version 2 of the toolkit. If you're not, or you need more information about versioning, early access, and upgrading, see [Early Access to Planet SDK (aka "V2")](v2_earlyaccess.md).
+=======
+You should be on some version 2 of the toolkit. If you're not, or you need to 
+>>>>>>> 2fa3e4e (first docs reorg)
+
+## Step 5: Sign on to your account
+
+Just as you log in when you browse to https://account.planet.com, you'll want to sign on to your account so you have access to your account and orders.
+
+## Step 6: Use the Toolkit to authenticate against the Planet servers
+
+```console
+$ planet auth init
+```
+
+You'll be prompted for the email and password you use to access [your account](https://account.planet.com).
+
+When you type in your password, you won't see any indication that the characters are being accepted. But when you hit enter, you'll know that you've succeeded because you'll see on the command line:
+
+```console
+Initialized
+```
+
+## Step 7: Creating your first order
+
+[*****TBW*****]
+
+The Planet SDK for Python allows Python developers to write software that makes
+use of the following Planet APIs:
+
+* [orders](https://developers.planet.com/docs/orders/)
+* [data](https://developers.planet.com/docs/data/) (not implemented)
+* [analytics](https://developers.planet.com/docs/analytics/) (not implemented)
+* [basemaps](https://developers.planet.com/docs/basemaps/) (referred to in the client as `mosaics`) (not implemented)
+
+The client modules within the Python library are asynchronous, which greatly
+speeds up many interactions with Planet's APIs. Support for asynchronous
+development is native to Python 3.6+ via the
+[`asyncio` module](https://docs.python.org/3/library/asyncio.html). A great
+resource for getting started with asynchronous programming in Python is
+https://project-awesome.org/timofurrer/awesome-asyncio. The Writings and Talks
```

Comment:
Yes! @jreiberkyle and I were just talking about putting in a link to some nice intro to async.